shell/common/shell.cc

```diff
@@ -1076,10 +1076,10 @@ void Shell::OnPlatformViewDispatchPlatformMessage(
 
   // The static leak checker gets confused by the use of fml::MakeCopyable.
   // NOLINTNEXTLINE(clang-analyzer-cplusplus.NewDeleteLeaks)
-  fml::TaskRunner::RunNowOrPostTask(
```

Comment:
Should we make this conditional on merged thread mode?

Reply:
Before merged thread mode, RunNowOrPost task would always force a task to be posted because it was not possible for platform channel responses to come from the UI thread - only from plugins on the platform thread. In merged thread mode, RunNowOrPostTask is immediately invoking the closure without scheduling a task - which means the dart event loop isn't worken up.

So all I'm doing here is removing the conditional behavior for merged thread mode which is always incorrect.